<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游戏设置</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>291</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>战机设定</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>HP：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>48</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QSpinBox" name="spHp">
      <property name="statusTip">
       <string>1</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>屏幕子弹数量：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QSpinBox" name="spBulSum">
      <property name="statusTip">
       <string>5</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>攻击力：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QSpinBox" name="spDam">
      <property name="statusTip">
       <string>2</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>子弹速度：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7" colspan="2">
     <widget class="QLineEdit" name="editBulSpeed">
      <property name="statusTip">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>起始位置：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>X：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="3">
     <widget class="QLineEdit" name="editX">
      <property name="statusTip">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Y：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7" colspan="2">
     <widget class="QLineEdit" name="editY">
      <property name="statusTip">
       <string>4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>223</width>
     <height>52</height>
    </rect>
   </property>
   <property name="title">
    <string>全局设定</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCheckBox" name="chcShowFps">
      <property name="statusTip">
       <string>8</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>显示FPS</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>刷新间隔(ms)：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spFps">
      <property name="statusTip">
       <string>9</string>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnRefresh">
      <property name="text">
       <string>刷新设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spHp</tabstop>
  <tabstop>spDam</tabstop>
  <tabstop>editX</tabstop>
  <tabstop>editY</tabstop>
  <tabstop>editBulSpeed</tabstop>
  <tabstop>chcShowFps</tabstop>
  <tabstop>spFps</tabstop>
  <tabstop>btnRefresh</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>on_set_main(QVariant,int)</slot>
 </slots>
</ui>
